packages/thumbprint-react/components/Avatar/index.module.scss

```diff
@@ -4,33 +4,33 @@
     position: relative;
 }
 
-.rootXsmall {
-    width: 20px;
-    height: 20px;
-    font-size: 8px;
-}
-
-.rootSmall {
+.rootXSmall {
```

Comment:
The class rename seems to not work nicely here:
https://deploy-preview-296--thumbprint.netlify.com/components/avatar/react/#example-extra-small

Haven't dug into this, but it may be related to Gatsby's CSS modules implementation. I've had to work around this bug in the past: https://github.com/gatsbyjs/gatsby/issues/5458